components/sp-solutions/org.wso2.carbon.sp.solutions.ei.analytics/widgets/HorizontalBarChart/webpack.config.js

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright (c)  2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var path = require('path');
+var webpack = require('webpack');
+var CopyWebpackPlugin = require('copy-webpack-plugin');
```

Comment:
should be `copyWebpackPlugin`

Reply:
Fixed in 53951238f84729f0c325006f8def576a3c702eda